src/SensitiveFloats.jl

```diff
@@ -84,6 +84,18 @@ function zero_sensitive_float{ParamType <: ParamSet}(
 end
 
 
+function zero_sensitive_float_array{ParamType <: ParamSet}(
+    ::Type{ParamType}, NumType::DataType, local_S::Int, d::Integer...)
+
+    sf_array = Array(SensitiveFloat{ParamType, NumType}, d)
+    for ind in 1:length(sf_array)
+        sf_array[ind] = zero_sensitive_float(ParamType, NumType, local_S)
+    end
+
+    sf_array
+end
```

Comment:
This makes sense, and I'll try to follow that guideline in the future.  But the PSF is a substantive change, and though I'm doing most of the work in a separate repo, in the interest of smaller PRs I think it makes sense to do some things like this first.